geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/IngestRasterCommand.scala

```diff
@@ -52,7 +52,8 @@ class IngestRasterCommand(parent: JCommander) extends Command with AccumuloPrope
       IngestRasterParams.FILE_PATH -> Some(params.file),
       IngestRasterParams.FORMAT -> Some(Option(params.format).getOrElse(getFileExtension(params.file))),
       IngestRasterParams.RASTER_NAME -> Some(params.rasterName),
-      IngestRasterParams.TIME -> Option(params.timeStamp)
+      IngestRasterParams.TIME -> Option(params.timeStamp),
+      IngestRasterParams.PARLEVEL -> Option(params.parLevel.toString)
```

Comment:
params.parLevel has default value 1. So it's safe to call params.parLevel.toString.